packages/synapse-react-client/src/components/Forum/DiscussionReply.tsx

```diff
@@ -40,6 +41,7 @@ export function DiscussionReply(props: DiscussionReplyProps) {
   })
 
   const isCurrentUserAuthor = reply.createdBy == currentUserProfile?.ownerId
+  const isReplyAuthorModerator = moderatorList?.includes(reply.createdBy)
```

Comment:
Why not just call `useGetModerators` here? 

Alternatively, if you are going to pass the precomputed `moderatorList`, you could just precompute the value of `isReplyAuthorModerator`. Kind of unusual to pass some data but still have to compute it here.